openslides/users/signals.py

```diff
@@ -63,40 +62,54 @@ def create_builtin_groups_and_admin(**kwargs):
         permission_dict['mediafiles.can_see'],
         permission_dict['motions.can_see'],
         permission_dict['users.can_see_name'], )
-    group_anonymous = Group.objects.create(name='Guests', pk=1)
+    group_anonymous = Group.objects.create(name='group_default', pk=1)
```

Comment:
Sorry: I think this line should be: `group_default = Group.objects.create(name='Default', pk=1)`